packages/components/src/Flex/Flex.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { CSSProperties, PropsWithChildren } from "react";
+// import chunk from "lodash/chunk";
+import classnames from "classnames";
+import { ColumnKeys, Direction, Spacing } from "./Flex.types";
+import styles from "./Flex.css";
+
+interface FlexProps extends PropsWithChildren {
+  /**
+   * Determine how the children gets laid out
+   *
+   * **Supported keys**
+   * - `"grow"` - Grows to the space available. If all children are set to
+   *   grow, then they'll have equal width.
+   * - `"shrink"` - Shrinks to the smallest size possible. Normally the size of
+   *   the child.
+   *
+   * By default, this will set every children to grow in equal widths.
```

Comment:
can just delete this, but that's super minor

```suggestion
```

Reply:
Sounds good. Committed.